tools/api-designer/README.md

```diff
@@ -0,0 +1,36 @@
+# RAPID ROCKS API DESIGNER
+
+An experimental designer for RAPID/ODATA APIs and RSDL-based services.
+
+![Screenshot](./screenshot.jpeg)
+
+## Build instructions
+
+### Build related tools
+
+This project depends on other tools in this repo. You have to manually build those tools first by going to their respective directories and following their build instructions (mainly a combination of `npm install` + `npm run build`):
+- [odataUri](../odataUri) (referenced as `odata-uri` in `package.json`)
+- [urlEditor](../urlEditor) (`odata-uri-editor`)
+- [visualModelEditor](../visualModelEditor) (`visual-model-editor`)
+- [rsdlEditor](../rsdlEditor) (`rsdl-editor`)
+
+**Note**: if you make modifications to any of those dependencies and rebuild and the changes don't take effect in the API designer, try:
+- uninstalling the dependency from the API designer (remove it from `package.json` and `node_modules` or run `npm uninstall`)
+- adding it back to `package.json`
+- running `npm run build`
+
+**TODO**: Consider organising this project as a monorepo to make dependency management of related tools easier. The current set up is quick hack.
+
+### Install dependencies
+
+Run:
+
+```
+npm install
+```
+
+### Launch the app locally
+
+```
+npm start
```

Comment:
After following the build instructions, `npm start` fails with
~~~sh
× Build failed.

@parcel/core: Failed to resolve 'process' from 'C:/git/odata-rapid/tools/rsdlEditor/dist/index.js' 

  C:\git\odata-rapid\tools\rsdlEditor\dist\index.js:19891:28
    19890 |     // Environment variable used to control console output
  > 19891 |     const verbose = typeof process != "undefined" && /\bparse\b/.test(process.env.LOG);
  >       |                            ^^^^^^^
    19892 |     let stackIDs = null;
    19893 |     function cutAt(tree, pos, side) {
~~~

Reply:
We found that adding the following property in the package.json file fixed the issue on my machine:

  "alias":{"process":{"global":"{}"}},

Reply:
I will push a commit with that fix.